add_connector.md

```diff
@@ -26,13 +33,21 @@ A connector is an integration to fulfill payments. Related use cases could be an
 
 Each Connector (say, a Payment Processor) could support multiple payment methods
 
-- **Cards :** Bancontact , Knet, Mada
-- **Bank Transfers :** EPS , giropay, sofort
-- **Bank Direct Debit :** Sepa direct debit
-- **Wallets :** Apple Pay , Google Pay , Paypal
-
-Cards and Bank Transfer payment methods are already included in Router. Hence, adding a new connector which offers payment_methods available in Router is easy and requires almost no breaking changes.
-Adding a new payment method (say Wallets or Bank Direct Debit) might require some changes in core business logic of Router, which we are actively working upon.
+- **Cards :** Visa, Mastercard, Bancontact , Knet, Mada, Discover, UnionPay
+- **Bank Redirects :** Ideal, EPS , Giropay, Sofort, Bancontact, Bizum, Blik, Interac, Onlnie Banking Czec Republic, Onlnie Banking Finland, Onlnie Banking Poland, Onlnie Banking Slovakia, Online Banking UK, Prezelwy24, Trustly, Online Banking Fpx, Online Banking Thailand
```

Comment:
```suggestion
- **Bank Redirects :** Ideal, EPS , Giropay, Sofort, Bancontact, Bizum, Blik, Interac, Online Banking Czec Republic, Online Banking Finland, Online Banking Poland, Online Banking Slovakia, Online Banking UK, Prezelwy24, Trustly, Online Banking Fpx, Online Banking Thailand
```

Also, Czech instead of Czec?